frontend/src/container/NotificationWall.jsx

```diff
@@ -216,7 +327,7 @@ export class NotificationWall extends React.Component {
                       }}
                     />
                   </div>
-                  {!notification.read && <i className='notification__list__item__circle fa fa-circle' />}
+                  {!notification.read && <i className='notification__list__item__circle fa circle' />}
```

Comment:
I didn't understand why you removed the `fa-`

Reply:
This is a typo, I'll add it back

Reply:
```suggestion
                  {!notification.read && <i className='notification__list__item__circle fa fa-circle' />}
```